src/openfl/display/DisplayObject.hx

```diff
@@ -1256,7 +1260,8 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 						
 					}
 					
-					__cacheBitmap.bitmapData = bitmapData;
+					__cacheBitmapData = bitmapData;
+					__cacheBitmap.bitmapData = __cacheBitmapData;
```

Comment:
could leave the `bitmapData` as the assigned value. not that it's measurable in any way really, but in theory local var access should be faster than field access